datafusion/sql/src/lib.rs

```diff
@@ -50,5 +50,9 @@ pub mod unparser;
 pub mod utils;
 mod values;
 
+#[deprecated(
+    since = "45.0.0",
+    note = "use datafusion_common::{ResolvedTableReference, TableReference}"
+)]
 pub use datafusion_common::{ResolvedTableReference, TableReference};
```

Comment:
I don't really know why this is publically re-exported but there is no reason to use it over what is in datafusion-common